### ISSUE TITLE
is it unsupported print with label mode?

### ISSUE THREAD

**Issue by purboindra** [andresperezmelo/print_bluetooth_thermal#39]
Hi, i wanna ask something, is this unsupported with label mode? cs, when i try to print with these package, everything workings fine with esc/pos mode. but, my code not working yet when i switch to label mode. 

my code completely follows the one in the example

thank you...

**Comment by andresperezmelo** [comment]
Currently not suitable for label printers, we are working on a plugin for label printers.

**Comment by niravp-cliqbux** [comment]
@andresperezmelo Any plan for Label Printing. Really in need for that. Thanks

**Comment by andresperezmelo** [comment]
Yes I'm working on it in my free time, my last days and I've been very busy, but I'll have to finish it soon.

**Comment by patelnirav48** [comment]
@andresperezmelo Thanks, I'm really in need for TSC Label Print labels.

**Comment by zhake555** [comment]
@andresperezmelo 

List<int> bytes = [];

        final profile = await CapabilityProfile.load();
        final generator = Generator(sharedPreference.getPaperSize[0], profile);
        bytes += generator.reset();

        int labelWidth = 40;
        int labelHeight = 30;
        int barcodeHeight = 80;
        int barcodeWidth = 2;
        int x = 80;
        int y = 60;

 var tspl = '''
        SIZE $labelWidth mm,$labelHeight mm
        TEXT $x, 30, 0, 1, 0, 0, "njknk"
        BARCODE $x,$y,"128",$barcodeHeight,1,0,$barcodeWidth,2,"$barcodeData"
        TEXT $x, 70, 1, 1, 1, 1, "hjjhjk"
        PRINT ${printConfig[0]}
        CLS
        END
        ''';

        bytes += generator.rawBytes(tspl.codeUnits);
        
        PrintBluetoothThermal.writeBytes(bytes);
        
        
        this prints TSPL but only barcode is printing. Text is not printing
